src/legacy/core_plugins/timelion/public/timelion_vis_fn.ts

```diff
@@ -84,7 +84,7 @@ export const getTimelionVisualizationConfig = (
       as: 'visualization',
       value: {
         visParams,
-        visType: name,
+        visType: 'timelion',
```

Comment:
because the visType that we register in `timelion/public/vis/index.ts` is called `timelion`, not `timelion_vis` as this function is.

Reply:
For further clarity, it would be more obvious if the values were bound by a common constant here and in `timelion/public/vis/index.ts`. How do you feel about it?